tests/temp/tempFileTestInternalApi.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+require_once(dirname(__FILE__) . "/../../interface/globals.php");
+
+
+// Looking into how to use the api locally via php calls (4 methods are shown below)
+
+
+// GUZZLE CALL OF API
+//  noted needed fix for when using localhost with a port when using the server
+//  does make me wonder if needing to use the server name for local api's will be very tough, sometimes impossible, to support, though
+//   (for example, if trying to run this through a php-fpm instance that does not include apache/nginx then not sure how it would work)
+use OpenEMR\Common\Http\oeHttp;
+
+error_log("DEBUG1: " . $_SERVER['HTTP_HOST'] . " : " . $GLOBALS['web_root']);
+$server = (preg_match('/^localhost:[0-9]+$/', $_SERVER['HTTP_HOST'])) ? 'localhost' : $_SERVER['HTTP_HOST'];
+$response = oeHttp::setLocalApiContext(true)->get($server . $GLOBALS['web_root'] . '/apis/api/facility');
+echo $response->body();
```

Comment:
I am not sure the the guzzle calls within php will work since the path to the host from within the server will not always match up with the path to the host from outside the server. For example, for the insane dev docker, from outside host use localhost:8081, but from inside host use localhost (note I added a work around above for this to work in my environment).

Reply:
Would it be better to use SERVER_NAME in this instance, instead of HTTP_HOST.  That will always be the same as it's dependent on the server configuration rather than the HTTP header sent with HTTP_HOST.

Reply:
I don't believe SERVER_NAME will contain the port in most cases.